externs/firebaseui-externs.js

```diff
@@ -326,20 +330,30 @@ firebaseui.auth.SignInOption = function() {};
  */
 firebaseui.auth.SignInOption.prototype.provider;
 
+/**
+ * Defines the sign-in option
+ * needed to configure the FirebaseUI federated sign-in widget.
+ *
+ * @interface
+ */
+firebaseui.auth.FederatedSignInOption = function() {};
+firebaseui.auth.FederatedSignInOption.prototype
+  = Object.create(firebaseui.auth.SignInOption);
+
```

Comment:
Replace with
```
/**
 * ...
 *
 * @interface
 * @extends {firebaseui.auth.SignInOption}
 */
firebaseui.auth.FederatedSignInOption = function() {};
```
You can use the same for `firebaseui.auth.EmailSignInOption` and `firebaseui.auth.PhoneSignInOption`

Reply:
Yeah seems fine.